### PR TITLE
Add links to backend repos, improve code dependency discovery

### PR DESCRIPTION
**Pull request by KastanDay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+* [Line-of-sight with Graph Attention Parser (LGAP)](https://gitlab.com/dprl/lgap-parser), visual parser to recognize chemical symbols from images.
+
+These 4 containers are built via [this Docker Compose](https://gitlab.com/dprl/dprl-alphasynthesis), and deployed via [this Helm chart](https://github.com/moleculemaker/chemscraper-helm-chart) in Kubernets.
 
 ## Development server
 
-For marwin.js license add the following lines to the ~/.npmrc file:
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.1.6.
+
+For marwin.js license add the following lines to your `~/.npmrc`:
```
Comment (bodom0015) [inline README.md:18]: Fixing my own minor typo from forever ago:
```suggestion
For marvin.js license add the following lines to your `~/.npmrc`:
```

**Review by bodom0015 — APPROVED** [review]
LGTM :+1: I fixed a minor typo that predated your changes

Thank you, @KastanDay! :tada:

File: README.md [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+## Backend repositories & related codebases
+
+Chemscraper frontend leverages these backends to parse chemical symbols and render the results.
+
+* [Chemscraper backend](https://gitlab.com/dprl/graphics-extraction/-/tree/prod), the primary entrypoint into backend code.
+* [Symbolscraper container](https://gitlab.com/dprl/symbolscraper-server), chemical parsing of parsing *'born digital'* PDFs.
+* [YOLO container](https://gitlab.com/dprl/yolov8sever) for localizing chemical symbols.
+* [Line-of-sight with Graph Attention Parser (LGAP)](https://gitlab.com/dprl/lgap-parser), visual parser to recognize chemical symbols from images.
+
+These 4 containers are built via [this Docker Compose](https://gitlab.com/dprl/dprl-alphasynthesis), and deployed via [this Helm chart](https://github.com/moleculemaker/chemscraper-helm-chart) in Kubernets.
```
Comment (bodom0015) [inline README.md:12]: Fixed one more minor typo:
```suggestion
These 4 containers are built via [this Docker Compose](https://gitlab.com/dprl/dprl-alphasynthesis), and deployed via [this Helm chart](https://github.com/moleculemaker/chemscraper-helm-chart) in Kubernetes.
```

**Comment by KastanDay** [comment]
@bodom0015 could you approve again? I merged your typo fix.

**Review by bodom0015 — APPROVED** [review]

**Comment by bodom0015** [comment]
Sure thing! Sorry, I didn't realize that a new commit would invalidate previous approvals

Good catch! :pray:

**Pull request merged**